### PR TITLE
chore(samples): add periodical instances deletion

### PR DESCRIPTION
**Pull request by FrodoTheTrue**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-compute/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by FrodoTheTrue** [comment]
@bcoe added instance deletion, as we discussed here: https://github.com/googleapis/nodejs-compute/pull/671

File: samples/test/util.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+/**
+ * Get VM instances created more than one hour ago.
+ */
+async function getStaleVMInstances() {
+  const projectId = await instancesClient.getProjectId();
+  const result = [];
+  const currentDate = new Date();
+  currentDate.setHours(currentDate.getHours() - 1);
```
Comment (summer-ji-eng) [inline samples/test/util.js:34]: Curious why minor 1 hour here?

File: samples/test/util.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+  const aggListRequest = instancesClient.aggregatedListAsync({
+    project: projectId,
+  });
+
+  for await (const [zone, instancesObject] of aggListRequest) {
+    console.log();
```
Comment (summer-ji-eng) [inline samples/test/util.js:41]: Not sure if this console.log is intention or not.

File: samples/test/util.js (hunk 1 above)
Comment (FrodoTheTrue) [reply samples/test/util.js]: Good point, for compute tests I switched to 3 hours.

Originally I got idea from nodejs-bigtable, there we have 1 hour: https://github.com/googleapis/nodejs-bigtable/blob/80022f846081a67c610870cec0e26f1da76d0dd5/samples/test/util.js#L44

Comment (summer-ji-eng) [reply samples/test/util.js]: Not sure the purpose. But except for this, in generall, looks good to me.

**Review by summer-ji-eng — APPROVED** [review]

**Review by bcoe — APPROVED** [review]

**Pull request merged**